src/magic_folder/config.py

```diff
@@ -1443,9 +1464,31 @@ def create_magic_folder(self, name, magic_path, author,
         :param unicode upload_dircap: the write-capability of the
             directory we upload data into.
 
+        :param int poll_interval: how often to scan for remote changes
+            (in seconds).
+
+        :param Optional[int] scan_interval: how often to scan for local changes
+            (in seconds).
+
         :returns: a MagicFolderConfig instance
         """
         valid_magic_folder_name(name)
+        if not isinstance(poll_interval, int) or poll_interval <= 0:
+            raise APIError(
+                code=http.BAD_REQUEST,
+                reason="Poll interval must be a positive integer (not '{!r}').".format(
+                    poll_interval
+                ),
+            )
+        if scan_interval is not None and (
+            not isinstance(scan_interval, int) or scan_interval <= 0
+        ):
+            raise APIError(
+                code=http.BAD_REQUEST,
+                reason="Scan interval must be a positive integer or null (not '{!r}').".format(
```

Comment:
`null` or `0` ? Now my confusion is complete 😸

Reply:
`null`